### PR TITLE
feature/login24

### PR DESCRIPTION
**Pull request by IgnasDav**

### CODE REVIEW

File: client/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import { AdminDashboardComponent } from './admin-dashboard/admin-dashboard.component';
+import { LoginComponent } from './login/login.component';
+import { FormsModule } from '@angular/forms';
+import { AppComponent } from './app.component';
+import { BrowserModule } from '@angular/platform-browser';
+
+
+const appRoutes: Routes = [
+  { path: '', redirectTo: '/login', pathMatch: 'full' },
+  { path: 'login', component: LoginComponent },
```
Comment (DariusPa) [inline client/src/app/app-routing.module.ts:12]:
```suggestion
  { path: '/login', component: LoginComponent },
```

File: client/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  { path: 'admin', component: AdminDashboardComponent },
```
Comment (DariusPa) [inline client/src/app/app-routing.module.ts:13]:
```suggestion
  { path: '/admin', component: AdminDashboardComponent },
```

File: client/src/app/app-routing.module.ts (hunk 1 above)
Comment (AudriusAnusauskas) [reply client/src/app/app-routing.module.ts:12]: I dont think there can be a slash used. If you use it, it gives error

File: client/src/app/app-routing.module.ts (hunk 2 above)
Comment (AudriusAnusauskas) [reply client/src/app/app-routing.module.ts:13]: same here

**Pull request merged**